### DISCUSSION TITLE
Expand the Floor Plan View in Dashboard

### DISCUSSION THREAD

**Discussion by pstoian** [ExperienceLovelace/ha-floorplan#16] [Q&A] [answered]
Hi,
First of all thank you for this great module. The install was well documented and easy to followed. Great job !!!

After creating my first floor plan using your sample, my question will be; it is there a way to expand the floor plan for the entire view of the dashboard (dimension noted as 2 in the below print screen) ? As now I see the floor plan with the with of a regular card (dimension noted as 1 in below print screen) 

Thank you.

![image](https://user-images.githubusercontent.com/11281042/104514849-8f827600-55b7-11eb-9bd0-6aaf92210195.png)

**Comment by exetico** [comment]
Hi,

I'll try and provide a good example tomorrow.

But here's a fast screenshot:
![Screenshot_20210113-230149](https://user-images.githubusercontent.com/3549445/104515713-74355c00-55f3-11eb-94a9-a4225e858106.jpg)

**Comment by townsmcp** [comment, marked as answer]
@pstoian to expand to full screen:

- Click the 3 vertical dots to the right of the screen
- Click ‘Edit Dashboard’
- Select your page
- Click the pencil to the right of the page title
- Select ‘Panel Mode’

**Reply by pstoian** [reply]
That worked - so the requirement is to have only on card on the dashboard and it will switch it to Panel Mode.

Thank you so much @townsmcp

**Reply by exetico** [reply]
That's great. Next time, I'll suggest you to mention that you're asking for a guide in the GUI - and not through YAML ☺️ My input works just fine, under the right circumstances. 😁 But I'll try and remember your question, so it can be added to the docs 🎉

**Reply by pstoian** [reply]
will do - you guys rock !!!! Thank you again.

**Reply by exetico** [reply]
Thank!

It's now added to the docs:
https://experiencelovelace.github.io/ha-floorplan/docs/how-to-handle-size-and-expand-floorplan/

While having your attention, please remember to update to [Home Assistant Core 2021.1.3 due to security-issues](https://www.home-assistant.io/blog/2021/01/14/security-bulletin/) (This has nothing to do with ha-floorplan 😄 )

**Comment by exetico** [comment]
For people finding this question on Google and so, please head over to our docs, where you'lll find guidance about this question:
https://experiencelovelace.github.io/ha-floorplan/docs/how-to-handle-size-and-expand-floorplan/
